src/components/common/Footer/components/FooterNav/styles.module.scss

```diff
@@ -0,0 +1,23 @@
+@import 'styles/variables';
+@import 'styles/mixins';
+
+.navigation {
+    list-style: none;
+    display: flex;
+    gap: 0 32px;
+    padding: 0;
+    margin: 0;
+}
+
+.link {
+    text-transform: capitalize;
+    @include hover;
+}
+
+@media screen and (max-width: 767px) {
```

Comment:
please use mobile first approach https://zellwk.com/blog/how-to-write-mobile-first-css/#:~:text=A%20mobile%2Dfirst%20approach%20to,uses%20min%2Dwidth%20media%20queries.